wgpu-types/src/lib.rs

```diff
@@ -580,6 +671,49 @@ pub struct RasterizationStateDescriptor {
     pub depth_bias_clamp: f32,
 }
 
+impl RasterizationStateDescriptor {
+    pub fn new() -> Self {
+        Self {
+            front_face: FrontFace::default(),
+            cull_mode: CullMode::default(),
+            clamp_depth: false,
+            depth_bias: 0,
+            depth_bias_slope_scale: 0.0,
+            depth_bias_clamp: 0.0,
+        }
+    }
+
+    pub fn front_face(&mut self, front_face: FrontFace) -> &mut Self {
+        self.front_face = front_face;
+        self
+    }
+
+    pub fn cull_mode(&mut self, cull_mode: CullMode) -> &mut Self {
+        self.cull_mode = cull_mode;
+        self
+    }
+
+    pub fn clamp_depth(&mut self, clamp: bool) -> &mut Self {
+        self.clamp_depth = clamp;
+        self
+    }
+
+    pub fn depth_bias(&mut self, bias: i32) -> &mut Self {
```

Comment:
let's combine all 3 depth bias parameters into a single call: they are related, and they mostly get set together